.ci/schedule-daily.groovy

```diff
@@ -22,7 +22,8 @@ pipeline {
       steps {
         updateBeatsBuilds(branches: ['main', '8.<minor>', '8.<next-patch>', '7.<minor>'])
         runWindowsBuilds(branches: ['main', '8.<minor>', '8.<next-patch>', '7.<minor>'])
-        runSmokeTests(branches: ['main'])
+        runSmokeTestsOs(branches: ['main', '8.<minor>', '8.<next-patch>', '7.<minor>'])
```

Comment:
Enable the smoke tests for supported Operating Systems on a daily basis for the current release branches and main.